home/tests/test_question_tag_model_functions.py

```diff
@@ -4,6 +4,25 @@
 from django.utils import timezone
 
 
+@pytest.fixture
```

Comment:
why is this soperfecial code move needed as part of this PR?

Reply:
I wanted to use this fixture for my tests, so i moved it out of this class instead of copy the entire function.

Reply:
Oh, you don't share fixture like that - to share fixtures you put then in a `conftest.py` file (look it up in the pytest docs).

But in any case you could have written that answer in the PR description and then I wouldn't have had to ask...